### PR TITLE
add 404 page for invalid urls.

### PR DESCRIPTION
**Pull request by Shubh1815**
Issue #35

### CODE REVIEW

**Comment by mann27** [comment]
Can you share a screenshot of the page?

File: src/components/404Page/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './404.css'
+
+export default function InvalidPage(){
+    return (
+        <div className='wrapper'>
+            <div>
+                <div className="status-code">404</div>
+                <p className="msg">Sorry! The Page You're Looking For Was Not Found.</p>
+            </div>
```
Comment (mann27) [inline src/components/404Page/index.js:10]: Also, add a go back to home page link.

Comment (Shubh1815) [reply src/components/404Page/index.js:10]: Okay, I'll add that

**Comment by mann27** [comment]
Make the 404 larger and with some intuitive font.

**Comment by Shubh1815** [comment]
is this okay???

**Comment by mann27** [comment]
Yes looks good 👍

**Review by mann27 — APPROVED** [review]

**Comment by mann27** [comment]
give @Shubh1815 75 points

**Comment by mann27** [comment]
give @Shubh1815 75 points

**Pull request merged**